.github/workflows/goreportcard.yaml

```
@@ -0,0 +1,52 @@
+name: Go Report Card
+
+on:
+  push:
+    branches:
+      - master
+      - main
+  pull_request:
+
+jobs:
+  goreportcard:
+    strategy:
+      matrix:
+        go: ['stable']
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Any reason to run this on all platforms? I believe the go report card is just source code scanning so the platform builds shouldn't be relevant

```suggestion
        os: [ubuntu-latest]
```

---

ditto